analysis/metrics.py

```diff
@@ -143,3 +144,157 @@ class AggregateMetrics:
     count_metrics: Optional[AggregateErrorMetrics] = None
     privacy_id_count_metrics: Optional[AggregateErrorMetrics] = None
     partition_selection_metrics: Optional[PartitionSelectionMetrics] = None
+
+
+# New dataclasses. They will be used as the output of Utility analysis in the
+# next PRs.
+
+
+@dataclass
+class MeanVariance:
+    mean: float
+    var: float
+
+
+@dataclass
+class ContributionBoundingErrors:
+    """Contains an error breakdown by types of contribution bounding.
+
+    Attributes:
+        l0: max_partition_contributed (aka l0) bounding error. The output of l0
+          bounding is a random variable for each partition.
```

Comment:
This one we know to be a Normally distributed when number of partitions is large, is it worth mentioning that?